src/lcm/example_models_stochastic.py

```diff
@@ -9,7 +9,7 @@
 
 
 def utility(consumption, working, health, partner, delta, gamma):  # noqa: ARG001
-    return jnp.log(consumption) + (gamma * health - delta) * working
+    return jnp.log(consumption + 1) + (gamma * health - delta) * working
```

Comment:
Why is this change necessary? The constraint `consumption > 0` should be handled elsewhere, if possible. Not clear to me why `consumption > -0.99999` is the easier thing to handle?

Reply:
Good point. Currently, we only have a proper constraint for `consumption < wealth`. I changed the grids of the model for `consumption` and `wealth` to both start at 1 (in principle, any positive number would work).